docs/_includes/floating-label.html

```diff
@@ -322,6 +322,160 @@ <h3>Floating Label with large select</h3>
+                <input class="textbox__control" type="tel" aria-label="area code + phone number"
+                  autocomplete="tel-national" aria-describedby="floating-phone-prefix-0"/>
+              </span>
+          </span>
+        </span>
+      </div>
+    </div>
+    {% highlight html %}
+    <span class="phone-input">
+        <span class="listbox-button listbox-button--form">
+          <button class="btn btn--form btn--split-start" aria-expanded="false" aria-haspopup="listbox" type="button"
+            data-listbox-button-prefix="Country: ">
+            <span class="btn__cell">
+              <span class="btn__text">
+                <svg class="flag flag--us" focusable="false" height="18" width="24" aria-hidden="true" data-country-code="1">
+                  <use href="#flag-us"></use>
+                </svg>
+              </span>
+              <svg class="icon icon--chevron-down-12" focusable="false" height="10" width="14" aria-hidden="true">
+                <use href="#icon-chevron-down-12"></use>
+              </svg>
+            </span>
+          </button>
+          <div class="listbox-button__listbox">
+            <div class="listbox-button__options" role="listbox">
+              <div class="listbox-button__option" role="option">
+                <span class="listbox-button__value">
+                  <svg class="flag flag--al" focusable="false" height="18" width="24" aria-hidden="true" data-country-code="355">
+                    <use href="#flag-al"></use>
+                  </svg>
+                  <span>Albania</span>
+                  <span>(+355)</span>
+                </span>
+                <svg class="icon icon--tick-16" focusable="false" height="10" width="14" aria-hidden="true">
+                  <use href="#icon-tick-16"></use>
+                </svg>
+              </div>
+              ...
+              <div class="listbox-button__option" role="option" aria-selected="true">
+                <span class="listbox-button__value">
+                  <svg class="flag flag--us" focusable="false" height="18" width="24" aria-hidden="true" data-country-code="1">
+                    <use href="#flag-us"></use>
+                  </svg>
+                  <span>United States of America</span>
+                  <span>(+1)</span>
+                </span>
+                <svg class="icon icon--tick-16" focusable="false" height="10" width="14" aria-hidden="true">
+                  <use href="#icon-tick-16"></use>
+                </svg>
+              </div>
+            </div>
+          </div>
+        </span>
+        <span class="field floating-label">
+          <label class="floating-label__label" for="field1"><span class="clipped">Area code + </span>Phone
+            Number</label>
+            <span class="textbox">
+              <span id="floating-phone-prefix-0">+1</span>
+              <input class="textbox__control" type="tel" aria-label="area code + phone number"
```

Comment:
@saiponnada As we already have a label element doing the job, we can remove aria-label here.